cmd/list.js

```diff
@@ -15,25 +15,23 @@
 
 const async = require('async');
 const init = require('../lib/init.js');
-const program = require('commander');
 const project = require('../lib/project.js');
 const user = require('../lib/user.js');
 const handleActionFailure = require('./../lib/util').handleCommandFailure;
 
-function list(command, cb) {
-  const options = init(command);
+function list(argv, cb) {
+  init(argv);
```

Comment:
Can we handle this `init` behavior centrally, as opposed to in every command? I realize it was like this before but maybe now would be a good time to split it up in order to separate concerns, improve testability, etc.

Reply:
Yeah, I was thinking the same but I'm not sure we can do it... I mean, we probably can one way or another but I believe `yargs` will introduce middlewares in the future. And that's a thing exactly for a middleware. But we can discuss it again during the refactoring.